src/components/Content/Breadcrumb.css

```diff
@@ -1,19 +1,21 @@
 @import url('../Helper/Helper.css');
 
 .breadcrumbs {
+  overflow: scroll;
```

Comment:
You could replace `overflow: scroll;` with `overflow-x: auto`, so that the vertical scrollbar is never shown, and the horizontal scrollbar is only shown when it's needed.

Reply:
done